src/components/UserBio/UserBio.js

```diff
@@ -1,37 +1,67 @@
-import React from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux'
+// icons
+import { FaTwitterSquare, FaLinkedin } from 'react-icons/fa';
 // components
 import UserFollow from '../UserFollow/UserFollow';
+import UpdateProfile from '../UpdateProfile/UpdateProfile';
 // styles
 import styles from './UserBio.module.scss';
-// images
-import profileImage from '../../assets/images/ksolo.jpg';
 
-const UserBio = () => (
-  <div className={ styles.profileHeader }>
-      <img src={ profileImage } alt="Kingsley Solomon" className={styles.profileImage} />
-      <span className={ styles.userNames }>Kingsley Solomon</span>
-      <div className={ styles.bioDiv }>
-        <div className={ styles.userBio }>
-          I am a cool Software Developer, who loves writing about my profession.
-          I want to let world know about the awesomeness of Tech
-        </div>
-        <div className={ styles.socialMedia }>
-          <div>
-            <a href="https://twitter.com"> <i className="fa fa-twitter faTwitter" /> </a>
-          </div>
-          <div>
-            <a href="https://facebook.com"> <i className="fa fa-facebook faFacebook" /> </a>
+class UserBio extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      updateProfileActive: false,
+    };
+  }
+
+  toggleUpdateProfile = (e) => {
+    e.preventDefault();
+    const { updateProfileActive } = this.state;
+    this.setState({ updateProfileActive: !updateProfileActive });
+  }
+
+  render() {
+    const { updateProfileActive } = this.state;
+    const { user } = this.props;
+    const { firstName, lastName, username, linkedin, twitter, bio, image } = user;
+
+    return (
+      <div className={styles.profileHeader}>
+        <img src={image || `https://ui-avatars.com/api/?name=${firstName}+${lastName}&background=C40018&color=fff&size=300`} alt={username} className={styles.profileImage} />
+        <span className={styles.userNames}>{`${firstName} ${lastName}`}</span>
+        <div className={styles.bioDiv}>
+          <div className={styles.userBio}>
+            {bio}
           </div>
-          <div>
-            <a href="https://linkedin.com"> <i className="fa fa-linkedin faLinkediN" /> </a>
+          {!linkedin && !twitter ? '' : <div className={styles.socialMedia}>
+            <ul>
+              {!twitter ? '' : <li>
```

Comment:
After updating my twitter field, clicking on the link in my profile should take me to Twitter, but it doesn't at the moment. This is what I get.
<img width="987" alt="screen shot 2018-10-25 at 10 03 52 am" src="https://user-images.githubusercontent.com/29521319/47489335-4dd24d00-d83e-11e8-9abf-4b8523e1eb8a.png">